### PR TITLE
Add definitions for reactcss

### PR DESCRIPTION
**Pull request by LKay**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -35,3 +35,4 @@ node_modules
 
 .sublimets
 .settings/launch.json
+yarn.lock
```
Comment (vvakame) [inline .gitignore]: could you make other pull request about this change?

Comment (LKay) [inline .gitignore]: Ok, I removed the this line.

**Comment by LKay** [comment]
Should I also make PR for `types-2.0` branch and make style adjustments if necessary or these changes will get populated by someone as bulk update of typings?

I believe the typings available through `typings` package are the ones published in `types-2.0` branch?

**Comment by vvakame** [comment]
cc/ @blakeembrey

**Comment by blakeembrey** [comment]
The `typings` CLI is pulling from master of DefinitelyTyped.

**Pull request merged**